client/src/modules/layouts/Contacts/ContactsPageTemplate.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import ContactData from '../../../models/ContactData';
+import CustomFieldSetting from '../../../models/customFields/CustomFieldSetting';
+import Table from '../../common/Table/Table';
+import NavBar from '../Navbar/Navbar';
+import * as styles from './ContactsPageTemplate.css';
+import { getTableData } from './ContactsService';
+import { isContactsDataValid } from './ContactsUtils';
+
+interface Props {
+  contacts: ContactData[];
+  domainCustomFields: CustomFieldSetting[];
+
+  loadAggregatedContacts(): void;
+}
+
+class ContactsPage extends React.Component<Props, object> {
+  public domainCustomFields: CustomFieldSetting[];
+  public modelName: string;
+  public noContactsMessage: string;
+  public oneContactHeader: string;
+  public moreThanOneContactsHeader: string;
+  public columns: any[];
+
+  public componentWillMount = () => {
```

Comment:
just use function declaration instead of arrow function and move `componentWillMount under `render()` method